src/server/network.ts

```diff
@@ -263,6 +264,19 @@ export type ResourceTiming = {
   responseStart: number;
 };
 
+export type IPAddressAndPort = {
+  ipAddress?: string;
```

Comment:
We can have an ipAddress without a port? Or a port without an ip? Or maybe these can be required and the whole object becomes optional?

Not important to me either way, I just want to make sure I understand. ip without a port hurts my head.

Reply:
Good point. I don't think I've seen a case where we only get one, so will make them both non-optional.